### DISCUSSION TITLE
vc-overlay-dynamic 动态对象属性

### DISCUSSION THREAD

**Discussion by bingolovess** [zouyaoji/vue-cesium#625] [Q&A]
vc-overlay-dynamic 动态对象属性如何动态删除数据，因为数据是随时可变的，初始化没问题，直接赋值，后面数据变化，其中有的数据不需要了怎么不影响其他的情况下，动态删除指定单个数据

**Comment by zouyaoji** [comment]
用ref的getOverlay方法 拿到动态对象，然后参考这一行：

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/components/overlays/dynamic/index.ts#L431


太忙了，建议你自己测试反馈。
